components/FlexibleForms/CheckboxField.tsx

```diff
@@ -41,7 +42,9 @@ const Field = ({
       className="govuk-fieldset"
       aria-describedby={hint && `${name}-hint`}
     >
-      <legend className="govuk-label lbh-label">{label}</legend>
+      <legend className="govuk-label lbh-label">
+        {label} {required && <span className="govuk-required">*</span>}
```

Comment:
i believe the correct markup for these spans is:

```
<span className="govuk-required">
    <span aria-hidden="true">*</span>
    <span className="govuk-visually-hidden">required</span>
</span>
```

that way they have a proper semantic meaning rather than just a visual one.